### PR TITLE
bug fix in executing backup urls - Windows

### PR DESCRIPTION
**Pull request by anandhu-eng**
In reference to issue: https://github.com/mlcommons/cm4mlops/issues/230

### CODE REVIEW

File: script/download-file/customize.py [hunk 1]
```diff
@@ -130,13 +136,14 @@ def preprocess(i):
 
         elif tool == "wget":
             if env.get('CM_DOWNLOAD_FILENAME', '') != '':
-                extra_download_options +=f" --tries=3 -O {q}{env['CM_DOWNLOAD_FILENAME']}{q} "
+                extra_download_options +=f" --tries=3 -O {q}{env['CM_DOWNLOAD_FILENAME']}{q} --no-check-certificate "
```
Comment (arjunsuresh) [inline script/download-file/customize.py:139]: --no-check-certificate should be used only when some ENV variable is set for security reasons.

Comment (anandhu-eng) [reply script/download-file/customize.py]: Added in commit https://github.com/GATEOverflow/cm4mlops/pull/118/commits/20bd0217fcac38aba9d5202d1f414deac88c234a.

Comment (arjunsuresh) [reply script/download-file/customize.py]: We can reuse this variable. 

https://github.com/GATEOverflow/cm4mlops/blob/mlperf-inference/script/download-file/_cm.json#L36

Comment (anandhu-eng) [reply script/download-file/customize.py]: Hi @arjunsuresh , through commit https://github.com/GATEOverflow/cm4mlops/pull/118/commits/848be8971a4aa475f6caa9073b62ab851f7767cb, `no-check-certificate` is turned on by default for `windows`.

**Review by arjunsuresh — APPROVED** [review]

**Pull request merged**